# Ломать — не строить
# Давайте немного поиграем в «багоюзеров».
#
# Вашему решению будет предоставлена функция func,
# которая принимает два позиционных параметра и производит с ними некую математическую операцию.
#
# Предложите вызов функции, который гарантированно породит ошибку внутри функции.
#
# Примечание
# Если ошибка произойдёт внутри функции, то она будет перехвачена и обработана.
# Если же она произойдет в вашем коде, то программа будет завершена с ошибкой.

func('1', None)
